netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -249,51 +272,40 @@ public void inboundDataShouldForwardToStream() throws Exception {
     // Create a data frame and then trigger the handler to read it.
     // Need to retain to simulate what is done by the stream.
     ByteBuf frame = dataFrame(3, false).retain();
-    handler.channelRead(this.ctx, frame);
+    handler.channelRead(ctx, frame);
     verify(stream).transportDataReceived(eq(content), eq(false));
   }
 
   @Test
-  public void createShouldQueueStream() throws Exception {
-    // Disallow stream creation to force the stream to get added to the pending queue.
+  public void createShouldBufferStream() throws Exception {
+    // Force the stream to be buffered.
     setMaxConcurrentStreams(0);
-    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream),
-        promise);
-
-    // Make sure the write never occurred.
-    verify(frameListener, never()).onHeadersRead(eq(ctx),
-        eq(3),
-        any(Http2Headers.class),
-        eq(0),
-        eq(Http2CodecUtil.DEFAULT_PRIORITY_WEIGHT),
-        eq(false),
-        eq(0),
-        eq(false));
+    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream), promise);
+    verifyNoMoreInteractions(ctx);
   }
 
   @Test
-  public void receivedGoAwayShouldFailQueuedStreams() throws Exception {
-    // Force a stream to get added to the pending queue.
+  public void receivedGoAwayShouldCompleteBufferedStream() throws Exception {
```

Comment:
maybe the name should be `...ShouldFailBufferedStream` or `...ShouldCancelBufferedStream`?